src/components-styled/data-sitemap.tsx

```diff
@@ -0,0 +1,143 @@
+                text={siteText.besmettelijke_personen.titel_sidebar}
+              />
+              <DataSiteMapLink
+                href="/landelijk/reproductiegetal"
+                text={siteText.reproductiegetal.titel_sidebar}
+              />
+              <DataSiteMapLink
+                href="/landelijk/sterfte"
+                text={siteText.sterfte.titel_sidebar}
+              />
+            </Box>
+            <Box>
+              <StyledHeader>
+                {siteText.nationaal_layout.headings.ziekenhuizen}
+              </StyledHeader>
+              <DataSiteMapLink
+                href="/landelijk/ziekenhuis-opnames"
+                text={siteText.ziekenhuisopnames_per_dag.titel_sidebar}
+              />
+              <DataSiteMapLink
+                href="/landelijk/intensive-care-opnames"
+                text={siteText.ic_opnames_per_dag.titel_sidebar}
+              />
+            </Box>
+            <Box>
+              <StyledHeader>
+                {siteText.nationaal_layout.headings.kwetsbare_groepen}
+              </StyledHeader>
+              <DataSiteMapLink
+                href="/landelijk/verpleeghuiszorg"
+                text={siteText.verpleeghuis_besmette_locaties.titel_sidebar}
+              />
+              <DataSiteMapLink
+                href="/landelijk/gehandicaptenzorg"
+                text={
+                  siteText.gehandicaptenzorg_besmette_locaties.titel_sidebar
+                }
+              />
+              <DataSiteMapLink
+                href="/landelijk/thuiswonende-ouderen"
+                text={siteText.thuiswonende_ouderen.titel_sidebar}
+              />
+            </Box>
+            <Box>
+              <StyledHeader>
+                {siteText.nationaal_layout.headings.vroege_signalen}
+              </StyledHeader>
+              <DataSiteMapLink
+                href="/landelijk/rioolwater"
+                text={siteText.rioolwater_metingen.titel_sidebar}
+              />
+              <DataSiteMapLink
+                href="/landelijk/verdenkingen-huisartsen"
+                text={siteText.verdenkingen_huisartsen.titel_sidebar}
+              />
+            </Box>
+            <Box>
+              <StyledHeader>
+                {siteText.nationaal_layout.headings.gedrag}
+              </StyledHeader>
+              <DataSiteMapLink
+                href="/landelijk/rioolwater"
+                text={siteText.nl_gedrag.sidebar.titel}
+              />
+            </Box>
+          </Box>
+        </Box>
+      </Box>
+    </Tile>
+  );
+}
+
+type DataSiteMapLinkProps = {
+  href: string;
+  text: string;
+};
+
+function DataSiteMapLink(props: DataSiteMapLinkProps) {
+  const { href, text } = props;
+  return (
+    <Link href={href}>
+      <StyledLink>
+        {text} <Chevron width="12px" height="12px" color={theme.colors.blue} />
+      </StyledLink>
+    </Link>
+  );
+}
+
+const StyledHeader = styled(Text).attrs({ as: 'span' })(
```

Comment:
Didn't we have an `InlineText` component which is the same as this Text with attrs?
```suggestion
const StyledHeader = styled(InlineText)(
```

Reply:
Indeed it is, I wasn't fully awake when I wrote this I think